e2e-tests/endpoints/kusama/blocks/10819306.json

```diff
@@ -2027,7 +2027,7 @@
             "info": {
                 "weight": "1177134000",
                 "class": "Normal",
-                "partialFee": "51999544"
+                "partialFee": "52002823"
```

Comment:
@TarikGul what's up with all these changes of `partialFee`?

Reply:
So here's my thought on this: `@substrate/calc` is part of older substrate legacy code when it comes to calculating fees. For a majority of our e2e-tests the partialFee we had before matched perfectly to the new partialFee we retrieve from the rpc method `payment::queryInfo`, but for the few cases that failed the accuracy was off by at most 4 decimal places (which is super tiny).  I'd be lying if I said I knew exactly what the issue was when it came to our intrinsic calculations. But, moving forward IMO the substrate rpc method is more reliable/accurate and if there was an discrepancy it was with the way we were either:

a) pulling the necessary, encodedLength, multiplier, perByteFee, or weights per runtime. 

b) Handling calculations and what runtime was using what fee calc version. 

Either way, moving off of `@substrate/calc`, and relying on the intrinsic calculations provided by the node seems like the right move given these results.